ctci/chapter1/1.8.py

```diff
@@ -0,0 +1,37 @@
+#1.8
+#Zero Matrix: Write an algorithm such that if 
+#an element in an MxN matrix is 0, its entire row and column are set to O.
+
+def zeroMatrix(m):
+    if (len(m) == 0 or len(m[0]) == 0):
+        return m
+
+    h = [False] * len(m)
+    w = [False] * len(m[0])
+
+    for i, col in enumerate (m):
```

Comment:
https://stackoverflow.com/a/16548714/12165177 seems promising to clean this up

Reply:
Ehh, I don't really like that, not really obvious what is going on

Reply:
Programming wasn't obvious before I could read, just get good :)